app.rb

```diff
@@ -1,8 +1,41 @@
 require 'sinatra/base'
+require './lib/player'
+require './lib/game'
+
 class RockPaperScissors < Sinatra::Base
+  before do
+    @game = Game.instance
+  end
+
```

Comment:
Well done for getting round the global variable, very good way of solving the issue.